mono/mini/interp/interp.c

```diff
@@ -7085,6 +7085,21 @@ interp_stop_single_stepping (void)
 	ss_enabled = FALSE;
 }
 
+/*
+ * interp_mark_stack:
+ *
+ *   Mark the interpreter stack frames for a thread.
+ *
+ */
+static void
+interp_mark_stack (gpointer thread_data, GcScanFunc func, gpointer gc_data, gboolean precise)
+{
+	if (!mono_use_interpreter)
```

Comment:
we shouldn't access this symbol. If we didn't use interpreter we shouldn't have these callbacks registered